robinandeer/chanjo-report
# Fix babel issue

### This PR adds | fixes:
- Fix #39 

### How to test:
- On master branch, try to launch the app with this command: `chanjo -d <database-connection-string> -r html`
- It should fail
- Try again from this branch


### Expected outcome:
- [ ] App startup should work from this branch

### Review:

This [version](https://semver.org/) is a:
